### PR TITLE
Add     "too_many_images_upload"

### PR DESCRIPTION
**Pull request by sam365724**

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "cancel": "Cancel",
     "preview": "Preview",
     "upload_image": "upload image",
+    "too_many_images_upload": "Maximum number of images to upload: 20. Please select fewer images.",
```
Comment (dessalines) [inline translations/en.json:47]: Don't hardcode that number, use either a generic message, or one with a count as shown below.

**Review by dessalines — CHANGES_REQUESTED** [review]

File: translations/en.json (hunk 1 above)
Comment (sam365724) [reply translations/en.json]: Good idea, will put the max into the message.

File: translations/en.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "cancel": "Cancel",
     "preview": "Preview",
     "upload_image": "upload image",
-    "too_many_images_upload": "Maximum number of images to upload: 20. Please select fewer images.",
+    "too_many_images_upload": "Exceeded maximum of {maxFormatted} images to upload. Please select fewer images.",
```
Comment (dessalines) [inline translations/en.json:47]: Did you look in this file to see how other plurals work?

Comment (dessalines) [reply translations/en.json:47]: "Exceeded maximum number of **1 images** to upload.

Please look in this file for how other plurals work with count.

Comment (sam365724) [reply translations/en.json:47]: I did look into it. Is the proposal from above fine with _plural?

Comment (sam365724) [reply translations/en.json]: @dessalines This ok? https://github.com/LemmyNet/lemmy-ui/pull/812/commits/692d8a2336c22e85ad7d0866da236b0886179f00

File: translations/en.json [hunk 3]
```diff
@@ -43,7 +43,9 @@
     "more": "more",
     "cancel": "Cancel",
     "preview": "Preview",
-    "upload_image": "upload image",
+    "upload_image": "upload image", 
+    "too_many_images_upload": "Exceeded maximum of {maxFormatted} image to upload. Please select only one image.",
+    "too_many_images_upload_plural": "Exceeded maximum of {maxFormatted} images to upload. Please select fewer images.",
```
Comment (dessalines) [inline translations/en.json:48]: Look at the other plural lines in this file. They use two braces.

**Review by dessalines — CHANGES_REQUESTED** [review]

**Review by dessalines — APPROVED** [review]

**Pull request merged**